resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -179,22 +131,85 @@ public DnsNameResolver(
      * @param nameServerAddresses the addresses of the DNS server. For each DNS query, a new stream is created from
      *                            this to determine which DNS server should be contacted for the next retry in case
      *                            of failure.
+     * @param minTtl
+     * @param maxTtl
+     * @param negativeTtl
+     * @param queryTimeoutMillis
+     * @param resolvedAddressTypes
+     * @param recursionDesired
+     * @param maxQueriesPerResolve
+     * @param traceEnabled
+     * @param maxPayloadSize
+     * @param optResourceEnabled
+     * @param hostsFileEntriesResolver
```

Comment:
Oops, I forgot to fill them.
It's better to do so if you want to upgrade to modern JDK as javadoc is now very picky.